lib/pbench/server/database/models/auth_tokens.py

```diff
@@ -6,12 +6,13 @@
 from pbench.server.database.database import Database
 
 
-class ActiveTokens(Database.Base):
+class AuthToken(Database.Base):
     """Model for storing the active auth tokens associated with a user."""
 
-    __tablename__ = "active_tokens"
+    # Table name is plural so that SQL statements read easier.
```

Comment:
I'm not sure these comments are really useful, especially now that you're making module names plural to match.

Reply:
Yeah...once we're over the transition, I don't think we'll need to justify the plural-ness of the names.

Reply:
I can certainly drop them, but it seems worth recording the reason for the difference.

Reply:
> it seems worth recording the reason for the difference

The difference between what and what?  That is, yeah, the module has one name, and the table has another name, and they are strikingly similar, but I don't _think_ that anyone but you is making the comparison.  So, if you want the comment to explain it, perhaps it would be worthwhile pointing out in the text what the table name is different _from_.  Otherwise, the reader goes, "Yeah, OK, it does, but why point that out?"

Reply:
Line 13?

Reply:
```
    # Table name is plural so that SQL statements read easier.
    __tablename__ = "auth_tokens"
```
If we rephrased that comment as, "The arbitrary string used below to designate the table name is made plural for external reasons which have no bearing on the Pbench Server source code", would it still seem worthwhile to include here?  That is, the name of the table is largely arbitrary, as it is hidden inside the SQLAlchemy object (right?).  And, the reason for making it plural is for the ease of human users interacting directly with the database and not for reasons present in the source code.  So, really, unless we're referring to the difference between the table name and the module name (which, at least in this particular case, there is none), I don't see anything worth pointing out _in the code_.

But, perhaps I'm missing something.

Reply:
Removed.